crypto/src/channel_encryption.cpp

```diff
@@ -16,11 +15,8 @@ std::vector<uint8_t> hexToBytes(const std::string& hex) {
 }
 
 template <typename T>
-ChannelEncryption<T>::ChannelEncryption(const std::string& key_path) {
-    // Lokid uses ed25519
```

Comment:
Is this comment still relevant?

Reply:
This is deleted

Reply:
Yeah, I'm asking if we should keep it instead

Reply:
Ah right, it was there to explain the function converting from ed25519 to curve25519, guess couldn't hurt to put back?